### ISSUE TITLE
Veeam Compatibility issue

### ISSUE THREAD

**Issue by omurozlu** [apache/cloudstack#8461]
##### ISSUE TYPE

 * Bug Report

##### COMPONENT NAME
Veeam B&R v12.0 Plugin

##### CLOUDSTACK VERSION
4.18.0.1
~~~

##### CONFIGURATION

##### OS / ENVIRONMENT

##### SUMMARY
We are trying connect Veeam B&R and getting the below error message. When will Veeam B&R 12 API be available?  
Error message we got;
"Request failed. (530) Veeam B&R API call unauthorized, please ask your administrator to fix integration issues."

##### EXPECTED RESULTS
We want to use Veeam B&R 12 
~~~

**Comment by weizhouapache** [comment]
@omurozlu 
please keep an eye on #8241

**Comment by omurozlu** [comment]
I cannot see this configuration in the backup.plugin.veeam.version UI section.

Am I looking for something in the wrong place?

<img width="1425" alt="Screenshot 2024-01-08 at 15 51 39" src="https://github.com/apache/cloudstack/assets/41322804/23dc9f14-7f30-410a-b0e3-2c64ec213366">

**Comment by weizhouapache** [comment]
> @weizhouapache I cannot see this configuration in the backup.plugin.veeam.version UI section.
> 
> Am I looking for something in the wrong place?
> 
> <img alt="Screenshot 2024-01-08 at 15 51 39" width="1425" src="https://private-user-images.githubusercontent.com/41322804/294904190-23dc9f14-7f30-410a-b0e3-2c64ec213366.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDQ3MTg4OTcsIm5iZiI6MTcwNDcxODU5NywicGF0aCI6Ii80MTMyMjgwNC8yOTQ5MDQxOTAtMjNkYzlmMTQtN2YzMC00MTBhLWIwZTMtMmM2NGVjMjEzMzY2LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAxMDglMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMTA4VDEyNTYzN1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWRjYzA4OWVkMWUxNzAyMzZjMTJjMTM1MDJlNjMxZmFkNDYwMDc4ODdjMTllMGYwMDM3YzVhNjlmZTc1ZjVkMjUmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.vVNlbSmO5xG55inD0vi2YY3RDbIRyJY63NyP9P6TS28">

@omurozlu 
The PR #8241 is not merged yet. It is still in testing

**Comment by omurozlu** [comment]
@weizhouapache ow sorry ty for answer

**Comment by omurozlu** [comment]
same error cloudstack v.19.0.0

**Comment by weizhouapache** [comment]
> same error cloudstack v.19.0.0

@omurozlu 
4.19.0.0 does not have the fix.
Please wait for 4.18.2.0 or 4.19.0.0
You can build the packages from source code as well

**Comment by omurozlu** [comment]
I probably thought wrong
<img width="455" alt="Screenshot 2024-02-07 at 01 56 06" src="https://github.com/apache/cloudstack/assets/41322804/2b1780f7-541e-43d9-a446-fc5f87b493c7">
I thought it was fixed

**Comment by weizhouapache** [comment]
> I probably thought wrong
> <img width="455" alt="Screenshot 2024-02-07 at 01 56 06" src="https://github.com/apache/cloudstack/assets/41322804/2b1780f7-541e-43d9-a446-fc5f87b493c7">
> I thought it was fixed

That is wrong. @omurozlu 
The commit was merged after 4.19.0.0

cc @shwstppr

**Comment by shwstppr** [comment]
@weizhouapache @omurozlu Release notes page should be fixed now. It gt merged during RCs and that resulted in showing up in changes.
@omurozlu sorry but for this fix will have to wait for 4.18.2 or 4.19.1

**Comment by shwstppr** [comment]
@omurozlu if you just need to test the compatibility for testing purpose, you may try the latest nightly package,
https://download.cloudstack.org/testing/nightly/

(should be available in the upcoming 2024-02-07 4.19/4.20 packages)

**Comment by weizhouapache** [comment]
> @omurozlu if you just need to test the compatibility for testing purpose, you may try the latest nightly package, https://download.cloudstack.org/testing/nightly/
> 
> (should be available in the upcoming 2024-02-07 4.19/4.20 packages)

thanks @shwstppr  !

@omurozlu 
as @shwstppr said, you can try the nightly build of 4.19/4.20. The packages of 2024-02-06 contain the bug fix.

**Comment by omurozlu** [comment]
> > @omurozlu if you just need to test the compatibility for testing purpose, you may try the latest nightly package, https://download.cloudstack.org/testing/nightly/
> > (should be available in the upcoming 2024-02-07 4.19/4.20 packages)
> 
> thanks @shwstppr !
> 
> @omurozlu as @shwstppr said, you can try the nightly build of 4.19/4.20. The packages of 2024-02-06 contain the bug fix.

ty for answer @weizhouapache  and @shwstppr
